src/commands/CopyModeratorsCommand.ts

```diff
@@ -29,8 +29,21 @@ export class CopyModeratorsCommand implements ICommand {
         }
         const fromRoomId = await this.client.resolveRoom(args[0]);
         const toRoomId = await this.client.resolveRoom(args[1]);
-        const fromPl: {"users"?: Record<string, any>} = await this.client.getRoomStateEvent(fromRoomId, "m.room.power_levels", "");
-        let toPl = await this.client.getRoomStateEvent(toRoomId, "m.room.power_levels", "");
+
+        let fromPl: { "users"?: Record<string, any> } = {}
+        try {
+            fromPl = await this.client.getRoomStateEvent(fromRoomId, "m.room.power_levels", "");
+        }
+        catch (error) {
+            throw Error(`Error fetching or processing power level event from room ${fromRoomId}: ${error.toString()}`)
+        }
+
+        try {
+            var toPl = await this.client.getRoomStateEvent(toRoomId, "m.room.power_levels", "");
```

Comment:
```suggestion
        let toPl: PowerLevelsEventContent;
        try {
            toPl = await this.client.getRoomStateEvent(toRoomId, "m.room.power_levels", "");
```

Avoid using `var` if you can, it doesn't conform to block scoping and tends to surprise you. In this case TypeScript should let you define `toPl` outside and it will always be set, because you are throwing.